ScarecrowJump/Source/Platform.cpp

```diff
@@ -0,0 +1,122 @@
+#include "Platform.h"
+#include "Player.h"
+
+bool Platform::checkCollision(Player& player)
+{
+	if (player.getIsFalling())
+	{
+		if (player.getBounds().top + player.getBounds().height <= m_sprite.getGlobalBounds().top + 3 && player.getBounds().top + player.getBounds().height >= m_sprite.getGlobalBounds().top - 3)
```

Comment:
Expose it into another helper function for more context, e.g. if (playerOnPlatformLevel())